internal/clientconn/listener_metrics.go

```diff
@@ -47,19 +49,41 @@ func NewListenerMetrics() *ListenerMetrics {
 			},
 			[]string{"error"},
 		),
+		requests: prometheus.NewCounterVec(
```

Comment:
Well, they do not belong there. Those were listener's metrics, and requests and responses are on a different level

Reply:
what about client conn?

Reply:
moved to clientconn.